src/activity_actor_definitions.h

```diff
@@ -95,6 +95,31 @@ class aim_activity_actor : public activity_actor
         void unload_RAS_weapon();
 };
 
+class autodrive_activity_actor : public activity_actor
+{
+    private:
+        vehicle *player_vehicle = nullptr;
+
+    public:
+        autodrive_activity_actor() {};
```

Comment:
```suggestion
        autodrive_activity_actor() = default;
```

Reply:
I managed to find a relevant thread about default constructors on stack overflow and now realize the errors of my ways.
https://stackoverflow.com/questions/20828907/the-new-syntax-default-in-c11